account/manager.go

```diff
@@ -798,6 +883,32 @@ func (m *manager) resumeAccount(ctx context.Context, account *Account, // nolint
 	return nil
 }
 
+// deriveFeeFromTx derives the transaction fee from a given transaction message.
+func (m *manager) deriveFeeFromTx(ctx context.Context,
+	tx *wire.MsgTx) (btcutil.Amount, error) {
+
+	// Input and output values are required to calculate the tx fee. However,
```

Comment:
nit: line length in several places in the file. You need to set up your IDE to use 8 spaces per tab character ("Tab size" in "Editor -> Code Style -> Go", tab "Tabs and Indents" in GoLand).

Reply:
Also, feel free to create a PR that adds the same `.editorconfig` from `lnd` which GoLand seems to use to auto-configure this.